packages/shared/src/components/modals/ReportPostModal.tsx

```diff
@@ -14,7 +14,7 @@ import {
 
 export interface Props extends ModalProps {
   postIndex: number;
-  post: Post;
+  post?: Partial<Post>;
```

Comment:
Why do we have to make this as a `Partial`type? I think the post should be required to report something.

Reply:
It should indeed, I must actually remake it to be type of CompanionPost/Post
As the CompanionPost has less data compared to a full post.